index.html

```diff
@@ -5,38 +5,49 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Christelle's portfolio</title>
-    <link rel="stylesheet" href="style.css" >
-    <link rel="stylesheet" href="media.css"/>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet "type="text/css">
+    <title>MyPortfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="media.css" />
+    <link rel="stylesheet" href="interact.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"
+        integrity="sha384-*********************" crossorigin="anonymous">
 </head>
 
 <body>
     <div class="main-wrapper">
         <header class="toolbar">
-            <div class="logo-place">
-                <a href="./index.html" class="my-logo hex">My Logo</a>
+            <div class="header-mobile">
+                <div class="logo-place">
+                    <a href="./index.html" class="my-logo hex">My Logo</a>
+                </div>
+                <nav>
+                    <!-- <figure class="menu">
+                        <img src="images/Icons/Union.png" alt="Icons" class="menuIcon" />
+                        <img src="images/Icons/Icon.png" alt="exit-icon" class="exit-icon">
+                    </figure> -->
```

Comment:
- [ ] Kindly remove the commented-out code from this file. Removing such codes makes your work look clean. 👍